awscli/customizations/generatecliskeleton.py

```diff
@@ -55,6 +55,9 @@ class GenerateCliSkeletonArgument(OverrideRequiredArgsArgument):
             'used with ``--cli-input-yaml``. If provided with the value '
             '``output``, it validates the command inputs and returns a '
             'sample output JSON for that command.'
+            'The generated JSON skeleton is not stable between versions '
```

Comment:
I think we need to add a space between `command.` and `The`. I think we can just add the space in the string above this line.